flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/streaming/connectors/elasticsearch/ElasticsearchApiCallBridge.java

```diff
@@ -40,12 +43,27 @@
 	/**
 	 * Creates an Elasticsearch {@link Client}.
 	 *
+	 * In comparison to {@link initClient}, this method creates a default {@link Client}.
```

Comment:
I don't think the `{@link initClient}` Javadoc link have the correct signature, does it?

For example, `{@link initClient}` should be `{@link #initClient(Map<String, String>, Function<Settings, TransportClient>)}` to actually link to that method.